tests/specs/close-db.js

```diff
@@ -5,10 +5,17 @@
         var indexedDB = window.indexedDB || window.webkitIndexedDB || window.mozIndexedDB || window.oIndexedDB || window.msIndexedDB;
 
         beforeEach(function (done) {
-            this.dbName = guid();
+            var spec = this;
+            spec.server = undefined;
+            if (!this.dbName) {
+                this.dbName = guid();
+                done();
```

Comment:
We want to add the `dbName` right away for the first test, but in the case of subsequent tests, we want to first delete the previous database (the old value of `dbName`) and then set the new value of `dbName`, etc.